lib/api/controllers/collectionController.js

```diff
@@ -153,18 +166,18 @@ class CollectionController {
       this.kuzzle.validation.specification[request.input.resource.index] === undefined ||
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 89 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)